mantidproject/mslice
# Hide error bars in cuts correctly

Description of work.
Added functionality to hide error bars in the plot options menu and the quick options menu
**To test:**


Open up mslice 
Load some data
Perform a cut plot
1. Open up the plot options menu and uncheck/check the error bars options for the relevant plot
2. Double-click a plot and uncheck/check the error bars option for the plot

The error bars should now be hidden from the legend and the actual plot


Fixes #372
